lib/widgets/highlight_text.dart

```diff
@@ -6,13 +6,19 @@ class HighlightText extends StatelessWidget {
   final String? searchWord;
   final TextStyle? style;
   final TextStyle? highlightStyle;
+  final int? maxLines;

```

Comment:
should add default

Reply:
these fields are nullable in Text.rich widget, it's fine imo

Reply:
no need